core/templates/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.directive.ts

```diff
@@ -394,10 +394,6 @@ angular.module('oppia').directive('stateGraphVisualization', [
             }
           };
           ctrl.$onInit = function() {
-            $scope.$on('redrawGraph', function() {
```

Comment:
Should there be a replacement for this?

Reply:
There were no broadcasts corresponding to triggering this so code would've never been hit anyway. It was being artificially through mock broadcast calls in the tests which I have since modified to be triggered on changing the window size instead.

Reply:
Ah ok makes sense, thanks for explaining!